### PR TITLE
Fix issues with dependencies breaking Travis CI

### PR DESCRIPTION
**Pull request by HashNotAdam**
Tests that check the number of attributes on a class in 1.8.7/ree still fail however I can't even compile Ruby that old to be able to look into it.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,1158 @@
+  AllCops:
```
Comment (plribeiro3000) [inline .rubocop.yml]: Was this file auto generated?

Comment (plribeiro3000) [inline .rubocop.yml]: I just want to be sure we don't loose our current style which should be based on github's one.

**Comment by plribeiro3000** [comment]
Btw, i can check the errors with `1.8.7` and if i'm not mistaken we should drop support for it already.

I will check on that and get back here soon.

**Comment by plribeiro3000** [comment]
Yeap. We can drop support for `1.8.7`. =)

File: .rubocop.yml (hunk 1 above)
Comment (HashNotAdam) [inline .rubocop.yml]: While the template was auto-generated, I created it with Code Climate and merged in anything that was customised in the previous file. For example, the default is to preference single-quoted strings but I've enforced double-quotes.

There are Rubocop validation errors but they are all related to the use of single-quotes.

**Comment by plribeiro3000** [comment]
Ok then. Do you want to fix your git credentials before i merge this?

**Comment by HashNotAdam** [comment]
Sorry @plribeiro3000, I don't understand

**Comment by plribeiro3000** [comment]
Your commit does not contain your personal mail does github does not recognize you as the author of those changes. Take a look here: https://help.github.com/articles/setting-your-email-in-git/

**Comment by HashNotAdam** [comment]
@plribeiro3000 Oh wow! I can't believe you're the first person to pick that up — rookie mistake.
Thanks

**Comment by plribeiro3000** [comment]
No worries. =)

I'm glad to help. 😉

**Comment by plribeiro3000** [comment]
Awesome!

**Comment by plribeiro3000** [comment]
Thank you very much for your contribution. =)

**Pull request merged**